.github/workflows/pull-request.yml

```diff
@@ -45,16 +48,10 @@ jobs:
         run: npm ci --no-audit --no-fund
 
       - name: Run Linting
-        run: npx turbo lint
-        env:
-          TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
-          TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+        run: npx turbo lint --filter=$TURBO_REF_FILTER
 
       - name: Run Prettier
-        run: npx turbo prettier
-        env:
-          TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
-          TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+        run: npx turbo prettier --filter=$TURBO_REF_FILTER
```

Comment:
will this exhibit the same Windows problem?

Reply:
No? This step runs only on Ubuntu